src/lib/query/search.ts

```diff
@@ -17,7 +17,7 @@ import { useDatabase } from "~/utils/db";
  */
 export async function searchMemory(
   params: QuerySearchRequest,
-): Promise<QuerySearchResponse> {
+): Promise<{ query: string, searchResults: SearchResults }> {
```

Comment:
This sounds to me like we should change QuerySearchResponse instead of defining an inline type